webpack.config.js

```diff
@@ -49,7 +50,7 @@ module.exports = {
     new CleanWebpackPlugin([path.join(__dirname, 'browser/dist')]),
     new HtmlWebpackPlugin({
       inject: false,
-      template: require('html-webpack-template'),
+      template: HtmlWebpackTemplate,
```

Comment:
This was throwing an ESLint error. I moved `require('html-webpack-template')` to the top of the page.